UI/platform-x11.cpp

```diff
@@ -27,8 +27,34 @@
 #include <locale.h>
 
 #include "platform.hpp"
+
+#ifdef __linux__
+#include <sys/socket.h>
+#include <string.h>
+#endif
+
 using namespace std;
 
+
+#ifdef __linux__
+void RunningInstanceCheck(bool &already_running)
+{
+	int uniq = socket(AF_LOCAL, SOCK_DGRAM, 0);
+
+	if (uniq == -1) {
+		blog(LOG_ERROR, "socket: %i", errno);
```

Comment:
It seems like we should just return here and startup normally, since there is no hope of bind succeeding at this point no reason to prompt the user with something that may be wrong either.
Please also make the blog more useful "Failed to check for running instance, socket: %d".

Reply:
Ok, will change that